Source/DafnyLanguageServer/DafnyLanguageServer.cs

```diff
@@ -11,9 +12,11 @@
 using OmniSharp.Extensions.LanguageServer.Server;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.Extensions.Options;
 
 namespace Microsoft.Dafny.LanguageServer {
   public static class DafnyLanguageServer {
+    public static List<string> LoadErrors = new();
```

Comment:
Expose this list as a property of type `IReadOnlyList<string>`.

Reply:
Then I have to assign the whole property at once in `LoadPlugins`, which mean I can't make `LoadErrors` readonly. Is that ok?

Reply:
You can implement the property so it uses a backing field. Making the external API readonly, but the internal mutable. For example you can do this:
```cs
private static List<string> loadErrors = new();
public static IReadOnlyList<string> LoadErrors => loadErrors;
```